cli/hzn.go

```diff
@@ -610,13 +616,13 @@ Environment Variables:
 			credToUse = cliutils.GetExchangeAuth(*exUserPw, *exServiceAddPolicyIdTok)
 		case "service removepolicy":
 			credToUse = cliutils.GetExchangeAuth(*exUserPw, *exServiceRemovePolicyIdTok)
-		case "business listpolicy":
```

Comment:
If user run `hzn exchange business listpolicy` does it go the the `deployment` case?

Reply:
Yes, it does